packages/server/graphql/mutations/helpers/notifications/MSTeamsNotifier.ts

```diff
@@ -43,6 +44,45 @@ const notifyMSTeams = async (
 }
 export type MSTeamsNotificationAuth = IntegrationProviderMSTeams & {userId: string}
 
+const createTeamPromptMeetingTitle = (meeting: Meeting) => `*${meeting.name}* is open 💬`
+
+const GENERIC_MEETING_TITLE = `Meeting Started 👋`
+
+const MeetingTypeTitleLookup: Record<MeetingTypeEnum, ((meeting: Meeting) => string) | string> = {
```

Comment:
-1 polymorphic values feel bad & clutter up the code down on L104, but I see the problem you're up against-- a lookup table is generally cleaner than a switch/case in javascript.

I'd prefer changing the generic case from a string constant into a thunk, just like you did for the actions! That way you don't have to duck-type the result of the lookup.

```ts
const createGenericTitle = () => `Meeting Started`
{action: createGenericTitle}
```

+1 camelCase. PascalCase is reserved for constructor-like entities (components, GraphQLObjects, client mutations. tbh I'm not sure why client mutations gets that treatment, but that's what facebook does so we all just started doing it like :sheep: )

+1 prefer to just provide `meeting.name`. treat function like the cops, don't give em any more than they need :police_officer:

Reply:
yeah, all good points, fixed!